src/wifi_hal_nl80211_utils.c

```diff
@@ -350,6 +369,11 @@ static const radio_interface_mapping_t static_radio_interface_map[] = {
     { 0, 0, "radio1", "wlan0"},
 #endif
 #endif
+
+#ifdef BANANA_PI_PORT
+    { 0, 0, "radio1", "wifi0"},
+    { 1, 1, "radio2", "wifi1"},
```

Comment:
Similar to the above comment, the first parameter should be having only one value as there is only one phy index

Reply:
Mentioned as above

Reply:
Removed above lines in latest commit